odoo/models.py

```diff
@@ -245,7 +246,6 @@ class BaseModel(object):
     _date_name = 'date'         # field to use for default calendar view
     _fold_name = 'fold'         # field to determine folded groups in kanban views
 
-    _needaction = False         # whether the model supports "need actions" (see mail)
```

Comment:
This has nothing todo with this task.
Please move this cleanup in a separate PR.